<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarketDataEditAssetDialog</class>
 <widget class="QDialog" name="MarketDataEditAssetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create/Edit asset</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout_5">
     <item row="3" column="1">
      <widget class="QLineEdit" name="assetID"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Unique asset identifier</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Asset name</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="assetName"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Asset type</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="assetType"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Asset symbol</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="assetSymbol"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="cmc">
      <attribute name="icon">
       <iconset resource="../../res/icons.qrc">
        <normaloff>:/icons/sources/cmc.png</normaloff>:/icons/sources/cmc.png</iconset>
      </attribute>
      <attribute name="title">
       <string>CoinMarketCap</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Filter</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="cmcFilterCoins"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Coin</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="cmcComboCoins"/>
       </item>
       <item row="2" column="1">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Source information</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="4" column="1">
           <widget class="QLineEdit" name="cmcBtcPrice"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Price (BTC)</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Circulating supply</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="cmcCirculatingSupply"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="cmcIdentifier"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Identifier</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Symbol</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="cmcSymbol"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gecko">
      <attribute name="icon">
       <iconset resource="../../res/icons.qrc">
        <normaloff>:/icons/sources/gecko.png</normaloff>:/icons/sources/gecko.png</iconset>
      </attribute>
      <attribute name="title">
       <string>CoinGecko</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Filter</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="geckoFilterCoins"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Coin</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="geckoComboCoins"/>
       </item>
       <item row="2" column="1">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Source information</string>
         </property>
         <layout class="QFormLayout" name="formLayout_4">
          <item row="2" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Circulating supply</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="geckoCirculatingSupply"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Price (BTC)</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="geckoBtcPrice"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Identifier</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="geckoIdentifier"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="geckoSymbol"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Symbol</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="exchanges">
      <attribute name="icon">
       <iconset resource="../../res/icons.qrc">
        <normaloff>:/icons/common/chain-link.png</normaloff>:/icons/common/chain-link.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Exchange map</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../res/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MarketDataEditAssetDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MarketDataEditAssetDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
